index.js

```diff
@@ -151,13 +151,22 @@ class TreeView extends React.PureComponent {
     return data.map(children => {
       const hasChildren = children[this.props.childrenKey] &&
         children[this.props.childrenKey].length > 0
-
+      var height = undefined
+      if (children.collapsed) {
+        if (this.props.collapsedItemHeightForLevel !== undefined) {
```

Comment:
Instead of doing that, we could make a way of defining a default function that resolves for that `height` prop

Reply:
Should to remove var collapsedItemHeight and always using function?

Reply:
That’s a good ideia I think